tests/bluetooth/bsim_bt/bsim_test_mesh/src/test_access.c

```diff
@@ -39,6 +44,82 @@ static uint8_t app_key[16] = { 0xaa };
 static uint8_t net_key[16] = { 0xcc };
 static struct bt_mesh_prov prov;
 
+/* Test vector for periodic publication tests. */
+static struct {
+	uint8_t period;
+	uint8_t div;
+	int32_t period_ms;
+} test_period[] = {
+	{ BT_MESH_PUB_PERIOD_100MS(5), 0, 500 },
+	{ BT_MESH_PUB_PERIOD_SEC(2),   0, 2000 },
+	{ BT_MESH_PUB_PERIOD_10SEC(1), 0, 10000 },
+	{ BT_MESH_PUB_PERIOD_SEC(3),   1, 1500 },
+	{ BT_MESH_PUB_PERIOD_10SEC(3), 3, 3750 },
+};
+
+/* Test vector for publication retransmissions tests. */
+static uint8_t test_transmit[] = {
+	BT_MESH_PUB_TRANSMIT(4, 50),
+	BT_MESH_PUB_TRANSMIT(3, 100),
+	BT_MESH_PUB_TRANSMIT(2, 200),
+};
+
+/* Test vector for canceling a message publication. */
+static struct {
+	uint8_t period;
+	uint8_t transmit;
+	uint8_t msgs;
+	int32_t sleep;
+	int32_t duration;
+} test_cancel[] = {
```

Comment:
I always pay attention to whether const is required first, but after thinking about it, in fact, running the script on a Linux device doesn't care about these few bytes.